src/main/java/com/book/mark/service/impl/CreateBookmarksServiceImpl.java

```diff
@@ -467,6 +467,27 @@ public void genSubmissionId() {
 				if(respBody != null) {
 					UnMarshaller unMarshaller = new UnMarshaller();
 					extObj = unMarshaller.unMarshallExternalIdResponse(respBody);
+					if(dataRequestPayload != null) {
+
+						try {
+							Region region = Region.US_GOV_WEST_1;
+							AwsSessionCredentials awsCreds = AwsSessionCredentials.create(
+									"AKIAXM6VM7LNUSMVFKRH",
+									"FEkOmLrH+dp61BcSfMl+VfwuzxqlmMk7WlhnRxVE",
+									"");
+							S3Client s3 = S3Client.builder()
+									.credentialsProvider(StaticCredentialsProvider.create(awsCreds))
+									.region(region)
+									.build();
+							PutObjectRequest objectRequest = PutObjectRequest.builder()
+									.bucket("ide-sandb-temp-microservice-bucket")
```

Comment:
There is a separate s3 bucket for each sandbox. How are you deploying this code for use in other sandboxes?